src/component/handlers/edit/editOnPaste.js

```diff
@@ -124,6 +126,14 @@ function editOnPaste(e: SyntheticClipboardEvent): void {
         );
         return;
       }
+    // Safari does not properly store text/html. Use the internalClipboard
+    // if present and equal to what is on the clipboard.
+    } else if (isSafari && internalClipboard &&
```

Comment:
I am not a big fan of browser target fixes like this. Whats the root cause of this problem ? can we solve with feature detection ?  I think we should avoid browser specific hacks in the library.